### ISSUE TITLE
[Adaptive Cards -Text Block] : 'Try It Yourself' link appear as Button when high contrast is enabled in 'Text Block' screen.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8543]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS Version: 23H2 (OS Build 25381.1)
URL: https://adaptivecards.io/explorer/TextBlock.html
Browser: Edge Dev Version 115.0.1880.3 (Official build) dev (64-bit)

**Pre-Requisite:**
Settings -> Accessibility -> Contrast themes ->Aquatic and Dessert modes. (High contrast Black/White mode).

**Repro Steps:**
1. Open the above URL in the edge dev browser.
2. Navigate to 'Text Block' under Card Elements and press enter key to activate it.
3. Navigate to the Try It Yourself link under example and observe the issue.

**Actual Result:**
'Try It Yourself' link appear as Button when high contrast is enabled in 'Text Block' screen.

**Expected Result:**
Correct role should be defined for the 'Community Call - October 2022' element and it should be link.

**Note:**
This issue is repro throughout the application for 'Try it Yourself' link.

**User Impact:**
When button controls appear as Button in high contrast modes, users who rely on that feature are blocked from information and functionality of the full user experience within the site/app.

“Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-vaibgu%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400980259288%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=TrcBaCyAPKg0WAQfdSZJP2i9pghMEDg59i2NeUUs5EQ%3D&reserved=0)“

### Screenshots

![WCAG4 3 1-'Try It Yourself' link appear as Button when high contrast is enabled in 'Text Block' screens](https://github.com/microsoft/AdaptiveCards/assets/95913748/4f44bb76-6cb0-4a3b-8b77-ef809b89c552)

![WCAG4 3 1-'Try It Yourself' link appear as Button when high contrast is enabled in 'Text Block' screen](https://github.com/microsoft/AdaptiveCards/assets/95913748/d1b4bd47-12c5-4c11-8966-a3a554441a2a)

![image](https://github.com/microsoft/AdaptiveCards/assets/95913748/a1ac5e35-0227-4555-aa92-e865a6a41d89)

### Card JSON

```JSON
NA
```
